src/wp-admin/upload.php

```diff
@@ -244,7 +244,8 @@
 					'<li>' . __( '<strong>Edit</strong> takes you to a simple screen to edit that individual file&#8217;s metadata. You can also reach that screen by clicking on the media file name or thumbnail.' ) . '</li>' .
 					'<li>' . __( '<strong>Delete Permanently</strong> will delete the file from the media library (as well as from any posts to which it is currently attached).' ) . '</li>' .
 					'<li>' . __( '<strong>View</strong> will take you to a public display page for that file.' ) . '</li>' .
-					'<li>' . __( '<strong>Copy URL to clipboard</strong> copies the URL for the media file to your clipboard.' ) . '</li>' .
+					'<li>' . __( '<strong>Copy URL</strong> copies the URL for the media file to your clipboard.' ) . '</li>' .
+					'<li>' . __( '<strong>Download file</strong> downloads the source media file to your computer.' ) . '</li>' .
```

Comment:
Based on [Rapid Tables](https://www.rapidtables.com/web/html/link/html-download-link.html)

```suggestion
					'<li>' . __( '<strong>Download file</strong> downloads a file from the server to the browser's directory on the local disk.' ) . '</li>' .
```

Reply:
Thanks @mukeshpanchal27 ! The suggestion is more general than the actual action, I would like something about which file being downloaded. The link points to the original file once uploaded.

So how about this? 
`<strong>Download file</strong> downloads the original file from the server to the browser's directory on the local disk.'`

Reply:
This is a little overly-technical for most users IMHO.

- Do we need to tell the user where the file is downloaded from? If a user clicks a download link, as far as they're concerned, they're downloading it from the page they're looking at.
- Do we need to tell a user where it specifically downloads to? They'll either know that because it's the default Downloads directory, or because they changed it, or because they just click/tap a downloads bar/notification on their device.
- Looking at this again, "your computer" is possibly a little presumptuous of us.

What about something like:

> <strong>Download file</strong> downloads the original media file to your device.

Is there anything the user will need to know that is missing from that?

Reply:
Thanks. It's good to have someone else review it who has more involvement in this area. WDYT?

Reply:
> <strong>Download file</strong> downloads the original media file to your device.

This looks better to me.

Reply:
Latest commit adjust help text to the suggestion by @costdev